articles/migrations/guides/extensibility-node12.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Migration Guide: Extensibility and Node 12"
+description: Learn about the Auth0 features affected by the Node.js v8 to Node.js v12 migration and review our recommendations for ensuring a smooth migration process.
+toc: true
+topics:
+  - migrations
+  - extensibility
+  - nodejs
+  - rules
+  - hooks
+  - custom-db
+  - custom-social-connections
+  - extensions
+contentType:
+  - concept
+  - how-to
+useCase:
+  - manage-accounts
+  - migrate
+---
+# Migration Guide: Extensibility and Node 12
+
+On December 31, 2019, [Node.js v8 went out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team no longer back-ports critical security fixes to this version. This _could_ expose your extensibility code to security vulnerabilities.
+
+As such, Auth0 is migrating from Node 8 to Node 12.
+
+**We will be ending support for the Node 8 runtime on June xx, 2020**. Tenants which have not already migrated to Node 12 will be migrated automatically. Automatic migrations will begin on April xx, 2020, and continue through June xx, 2020. Automatic migrations are happening in cohorts: 
+
+* Free tier tenants will be migrated beginning on April xx, 2020
```

Comment:
Needs dates, migration plan updates.

Reply:
I spoke with @charlesjohnson, and I think we'd like to remove this entire section around dates. We are going to make Node 12 available and let tenants migrate to it on their own time. At this time, we have no plans to public official dates saying when Node 8 will be retired.